core_backend/app/question_answer/speech_components/external_voice_components.py

```diff
@@ -57,16 +52,12 @@ async def transcribe_audio(audio_filename: str) -> str:
 
 async def generate_tts_on_gcs(
```

Comment:
Calling the function `generate)tts_on_gcs` is confusing. You should use another name for the function